modules/Microsoft.MachineLearningServices/workspaces/.test/parameters.json

```diff
@@ -34,14 +34,14 @@
         "discoveryUrl": {
             "value": "http://example.com"
         },
-        "encryptionIdentity": {
+        "cMKUserAssignedIdentityResourceId": {
```

Comment:
For other resources we provide a dedicated test through encr.parameters.json. Do we want to keep the same for consistency?

Reply:
We can. For some resources the test actually needed to be a different file, but not for all. Will adjust